src/main/java/com/r307/arbitrader/DecimalConstants.java

```diff
@@ -7,4 +7,9 @@
     private DecimalConstants() {
         // this method intentionally left blank
     }
+
+    //An intermediate scale is necessary to limit rounding errors when queueing BigDecimal.divide calls
+    public static int getIntermediateScale(int scale) {
```

Comment:
It's not really a constant, is it? ;)

It can stay here but I'd ask whether it would be sufficient to use `BTC_SCALE` instead or just put it on `TradeVolume`. It appears to only be used there and the subclass `EntryTradeVolume`.